controllers/metal3.io/hostfirmwaresettings_controller.go

```diff
@@ -182,6 +182,18 @@ func (r *HostFirmwareSettingsReconciler) updateHostFirmwareSettings(currentSetti
 	// get or create a firmwareSchema to hold schema
 	firmwareSchema, err := r.getOrCreateFirmwareSchema(info, schema)
 	if err != nil {
+		// forbidden error can be considered harmless when CleaningModeDisabled and ns terminating
+		if k8serrors.IsForbidden(err) {
```

Comment:
Is logging this error actually a problem? There's no finalizer on HostFirmwareSettings so it's just going to be deleted immediately upon the namespace being deleted I think. Even if it weren't, this isn't blocking anything important from proceeding as far as I can tell.